### DISCUSSION TITLE
how to configure editor panel's overlay on lose focus

### DISCUSSION THREAD

**Discussion by phuctt10kn** [NvChad/NvChad#2924] [Q&A] [answered]
i misstyped somethings and now editor panel is overlayed when i switch focus as below image, i want to keep this config but i cant find it out. pls help 
![image](https://github.com/user-attachments/assets/47a9610c-5b08-4bc5-a729-c296be1a0b56)

**Comment by siduck** [comment]
what do u mean by panel's overlay? that floating terminal ?

**Reply by phuctt10kn** [reply]
I mean the editor panel becomes a bit darker when I focus on the terminal or nvim-tree

**Reply by siduck** [reply, marked as answer]
that wont happen by default in neovim, some plugin's doing it i think, are u using shade.nvim?

**Reply by phuctt10kn** [reply]
Shade.nvim really lies in my config 🤣, thanks for ur help, I will check their docs

**Reply by siduck** [reply]
@phuctt10kn  you dont need to check the docs! that plugin does that job only i.e dim inactive windows

**Reply by phuctt10kn** [reply]
It simply needs to be loaded. I've updated its lazy load config.
